lib/mruby/lockfile.rb

```diff
@@ -0,0 +1,73 @@
+autoload :YAML, 'yaml'
+
+module MRuby
+  autoload :Source, 'mruby/source'
+
+  class Lockfile
+    class << self
+      def enable
+        @enabled = true
+      end
+
+      def disable
+        @enabled = false
```

Comment:
You should check this that it's already disabled

Reply:
What is the concrete problem with multiple `MRuby::Lockfile.disable`? I think idempotent is not bad.